lib/signaling/participant.js

```diff
@@ -12,14 +12,25 @@ ParticipantSignaling States
     | connecting |---->| connected |---->| disconnected |
     |            |     |           |     |              |
     +------------+     +-----------+     +--------------+
-
+                           | ^                    ^
```

Comment:
I think we need an arrow from `connecting` to `reconnecting`.

Reply:
@makarandp0 "connecting" immediately transitions to "connected", as soon as the ParticipantSignaling object is created. So there is no possibility of the transition you are talking about here.

Reply:
@makarandp0 I need to remove "reconnecting" from the list of states that are allowed to transition from "connecting".